.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: node_js
   - 0.10
+env:
```

Comment:
revert?

Reply:
Needed to ensure build passes on travis ci (the hand made parsing of the java version we have doesn't work, so build is currently failing on travis because of this, cf for example: https://travis-ci.org/github/youtube/spfjs/builds/656613239, a maintenance CL not merged because of this). 

Rationale for this ENV option: We control travis environment, so we can skip the java version check (that java version check is done to ensure closure can work; If there was anyhow a different version of java used, closure build would fail).